app/browser-sync.ts

```diff
@@ -2,7 +2,9 @@ import * as ab from 'asyncblock';
 import { ProtractorBrowser } from 'protractor';
 import { ILocation, ISize, IWebDriverOptionsCookie, Options, TargetLocator, Window } from 'selenium-webdriver';
 
+import {ElementFinderSync} from './element-finder-sync';
```

Comment:
nit: spacing on this and the other import if we want to keep with the code style :)